app/views/admin/clients/_edit_case_modal.html.erb

```diff
@@ -19,7 +19,13 @@
 
           <%= f.input :gps_required, as: :select, include_blank: false, wrapper: :horizontal_form %>
 
-          <%= f.input :court, wrapper: :horizontal_form %>
+          <%= f.input :court, wrapper: :horizontal_form, input_html: { list: "courtlist" } %>
+
+          <datalist id="courtlist">
```

Comment:
We're using the simple_form library for our forms, & this doesn't show up in the UI for me - take a look at the facilities field as an example of how to do a select fields with a collection like this. 

In particular, if you pull master you'll see an example of how to make the selected value be the name field instead of the default id

Reply:
The datalist tag should be hidden, but create a dropdown of suggestions for filling in the text field. 

I think it works in most browsers but not safari apparently. Here's what it looks like for me (with an empty text input)
![court-dropdown-new](https://user-images.githubusercontent.com/4087850/54077431-355f4e00-4286-11e9-9cad-a0cfe05b1e7f.png)
![court-dropdown-1](https://user-images.githubusercontent.com/4087850/54077432-355f4e00-4286-11e9-8812-e123640a8301.png) and here's a live example from w3schools https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_datalist.

My original attempt was going to be making it free text while we figured out what the options should be, but I'll see if I can get a complete list from Atara and make this a little stricter.

Reply:
Ahh, interesting, nice! I'd say this is fine as-is as well, as long as we're sure to explain to Atara what to expect.